Sources/TestingMacros/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+cmake_minimum_required(VERSION 3.28)
+
+if(POLICY CMP0157)
+  cmake_policy(SET CMP0157 NEW)
+endif()
+
+project(TestingMacros
+  LANGUAGES Swift)
+
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
+
+add_compile_definitions(_CRT_SECURE_NO_WARNINGS)
+include(FetchContent)
```

Comment:
FetchContent is meant to do a find_package, but as far as I can tell when I've tested it that's never worked. So ideally this would do a find_package and then FetchContent if that fails. In either case the dependencies should use the prefix SwiftSyntax:: so that it works whether find_package or FetchContent is used (swift-syntax adds SwiftSyntax:: aliases, so they are there all the time).

Reply:
Sure, we can do that.